src/smc-webapp/landing_page.cjsx

```diff
@@ -676,27 +680,59 @@ exports.LandingPage = rclass
             forgot_password_success = {@props.forgot_password_success}
         />
 
+    # this is an info blob on the landing page, clarifying to the user that "free" is a perpetual trial
+    render_trial_info: ->
+        return null if not (@props.customize?.is_commercial ? true)
```

Comment:
This is wrong; this is not how redux props works.  It should be @props.is_commercial.  I'll fix this.